docs/data/date-pickers/custom-views/custom-views.md

```diff
@@ -0,0 +1,73 @@
+---
+productId: x-date-pickers
+title: Date and Time Pickers - Custom views
+components: PickersCalendarHeader, PickersRangeCalendarHeader, PickersMonth, PickersYear
```

Comment:
```suggestion
components: MonthCalendar, PickersCalendarHeader, PickersRangeCalendarHeader, YearCalendar
```
Both of these do not exist.
https://deploy-preview-13321--material-ui-x.netlify.app/x/api/date-pickers/pickers-month/
https://deploy-preview-13321--material-ui-x.netlify.app/x/api/date-pickers/pickers-year/

If we are exporting `PickersMonth` and `PickersYear`, they should probably also get API pages, but that's why I'm advocating for not exporting. 🙈

Reply:
It's a leftover from when the slot was replacing `PickersMonth` instead of just the button
I'll fix this list :+1: